EasyReflectometryApp/Gui/Pages/Sample/MultiLayerTable.qml

```diff
@@ -17,7 +17,8 @@ EaComponents.TableView {
         property int layersIndex: ExGlobals.Constants.proxy.model.currentLayersIndex + 1
 
         xml: ExGlobals.Constants.proxy.model.layersAsXml
-        query: `/root/item[${itemsTable.currentIndex + 1}]/layers/item`
+        // query: `/root/item[${itemsTable.currentIndex + 1}]/layers/item`
+        query: `/data/item/layers`
```

Comment:
This means that the layer editor won't change when you change the item selection